modelcenter/PP-LCNet/introduction_en.ipynb

```diff
@@ -0,0 +1,308 @@
+    "\n",
+    "| large-kernel Location       | Top-1 Acc(\\%) | Latency(ms) |\n",
+    "|:--:|:--:|:--:|\n",
+    "| 1111111111111     | 63.22           | 2.08         |\n",
+    "| 1111111000000     | 62.70           | 2.07        |\n",
+    "| <b>0000001111111<b>     | <b>63.14<b>           | <b>2.05<b>         |\n",
+    "\n",
+    "Experiments show that a larger convolution kernel placed in the middle and back of the network can achieve the accuracy of placing in all positions, and at the same time, faster inference speed can be obtained. PP-LCNet finally chose the third row of the table.\n",
+    "\n",
+    "\n",
+    "#### 4.2.4 Use a Larger 1x1 Convolution Layer After the GAP\n",
+    "After GoogLeNet, the classification layer is often directly adopted after the GAP(Global-Average-Pooling). However, in lightweight networks, this will lead to no further integration and processing of features extracted after the GAP. If a larger 1x1 convolution layer (equivalent to FC layer) is used thereafter, the features after GAP will not be directly through the classification layer, but the fusion is first carried out, and the fusion features are classified. This can greatly improve the accuracy without affecting the model inference speed. BaseNet obtained PP-LCNet by improving the above four aspects. The following table further illustrates the impact of each scenario on the results:\n",
+    "\n",
+    "| Activation | SE-block | Large-kernel | last-1x1-conv | Top-1 Acc(\\%) | Latency(ms) |\n",
+    "|:--:|:--:|:--:|:--:|:--:|:--:|\n",
+    "| 0       | 1       | 1               | 1                | 61.93 | 1.94 |\n",
+    "| 1       | 0       | 1               | 1                | 62.51 | 1.87 |\n",
+    "| 1       | 1       | 0               | 1                | 62.44 | 2.01 |\n",
+    "| 1       | 1       | 1               | 0                | 59.91 | 1.85 |\n",
+    "| <b>1<b>       | <b>1<b>       | <b>1<b>               | <b>1<b>                | <b>63.14<b> | <b>2.05<b> |\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 5. Notes\n",
+    "The delay test of PP-LCNet series model at the Intel CPU was completed based on the Intel Xeon Gold 6148. PaddlePaddle was used in the inference framework, MKLDNN is turned on, and the batch size was 1 and the number of threads was 10."
```

Comment:
-> The inference speed of PP-LCNet series model were tested at Intel Xeon Gold 6148 CPU with PaddlePaddle inference framework, and MKLDNN was turned on, the batch size was 1 and the number of threads was 10.